### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yjshen324**

### CODE REVIEW

**Comment by robinshine** [comment]
Great for the rest

**Comment by robinshine** [comment]
a is great

File: test.java [hunk 1]
```diff
@@ -1 +1,2 @@
 a
+b
```
Comment (robinshine) [inline test.java:2]: works like a b

Comment (robinshine) [inline test.java]: Maybe a and b should be on same line?

Comment (yjshen324) [reply test.java]: Check my new change dude

File: test.java [hunk 2]
```diff
@@ -1,6 +1,3 @@
-111
-222
-333
-444
-555
-666
+//a
+//b
+a+b
```
Comment (robinshine) [inline test.java:3]: Oh shit, I mean "a b"

File: Dude.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+public class Dude {
+	public static void main(String[] args) {
+		System.out.println("hello world");
```
Comment (robinshine) [inline Dude.java:3]: Please use logger.info("hello world") here

File: Dude.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 public class Dude {
 	public static void main(String[] args) {
-		System.out.println("hello world");
+		logger.info("hello world");
```
Comment (robinshine) [inline Dude.java:3]: may be logger level "warn" is more appropriate here

**Comment by robinshine** [comment]
Why are you changing? Revert back!

File: test.java [hunk 5]
```diff
@@ -1,3 +1,5 @@
 //a
 //b
-a b
+a&b
+c&d
+e&f
```
Comment (robinshine) [inline test.java:5]: Change '&' of all of these to space

**Pull request closed without merging**